drivers/led/pca9533.c

```diff
@@ -65,9 +65,9 @@ struct pca9533_data {
  */
 static uint8_t ms_to_psc(uint32_t period_ms)
 {
-	uint32_t tmp = (period_ms * 152U + 500U) / 1000U;
+	int32_t tmp = (period_ms * 152U + 500U) / 1000U;
```

Comment:
should probably have
```
	int32_t tmp = (int32_t)((period_ms * 152U + 500U) / 1000U);
```
or will get a future warning on type assignment

Reply:
I am not familiar with the warning.
Can you share a link please?